usage/html.go

```diff
@@ -108,14 +111,30 @@ func htmlHelpAction(ctx *cli.Context) error {
 
 	// Subcommands
 	for _, cmd := range ctx.App.Commands {
-		if err := htmlHelpCommand(ctx.App, cmd, path.Join(dir, cmd.Name)); err != nil {
+		if err := htmlHelpCommand(ctx.App, cmd, path.Join(dir, cmd.Name), report); err != nil {
 			return err
 		}
 	}
+
+	// report
+	repjson := path.Join(dir, "report.json")
+	rjw, err := os.Create(repjson)
```

Comment:
This will override a previous report. If you think this can be an issue you can use utils.WriteFile() that will ask the user for consent.

Reply:
Cool. I think I will stick with what's there now. The default behavior of the HTML help is to overwrite existing files.